ts/live-compositor/src/components/Tiles.ts

```diff
@@ -63,7 +63,7 @@ function sceneBuilder(props: TilesProps, children: SceneComponent[]): Api.Compon
     children: children.map(sceneComponentIntoApi),
     width: props.width,
     height: props.height,
-    background_color_rgba: props.backgroundColor,
+    background_color_rgba: props.backgroundColor && intoApiRgbaColor(props.backgroundColor),
```

Comment:
unrelated to the node issue (it does not seem to be worth to create separate PR)